openmetrics-api/src/test/java/io/airlift/openmetrics/TestMetricExpositions.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.openmetrics;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.openmetrics.types.BigCounter;
+import io.airlift.openmetrics.types.Counter;
+import io.airlift.openmetrics.types.Gauge;
+import io.airlift.openmetrics.types.Info;
+import io.airlift.openmetrics.types.Summary;
+import org.testng.annotations.Test;
+
+import java.math.BigInteger;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestMetricExpositions
```

Comment:
nit: I think test would me more readable if you use multiline literals instead concatenating.

Also use of production class `ExpositionHelper` to generate expected result is antipattern.